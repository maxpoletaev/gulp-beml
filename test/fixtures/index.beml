<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Hello world</title>
  </head>
  <body block="b-page">
    <div block="b-block" mod="size:large">
      <div elem="elem">
        <div elem="elem-name" mix="block:b-mix, elem:elem">Hello world</div>
      </div>
    </div>
  </body>
</html>
