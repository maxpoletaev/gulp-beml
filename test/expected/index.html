<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Hello world</title>
  </head>
  <body class="b-page">
    <div class="b-block b-block_size_large">
      <div class="b-block__elem">
        <div class="b-block__elem-name b-mix__elem">Hello world</div>
      </div>
    </div>
  </body>
</html>
